gradle.properties

```diff
@@ -14,7 +14,7 @@ jetty8_version=8.1.8.v20121106
 jetty8_servlet_api_version=3.0.1
 jetty9_version=9.2.22.v20170606
 jetty93_version=9.3.20.v20170531
-jetty94_version=9.4.6.v20170531
+jetty94_version=9.4.7.v20170914
```

Comment:
I've been using `jetty94Version=9.4.8.v20171121` in a local project for a couple of weeks, and it seems to be working OK.  Was released Nov 2017.  Would you mind testing that and if OK updating the PR?

Reply:
I notice that this is based of someone else's branch that you probably won't have access-to.  To save messing with another PR I'll just merge this then bump the version up again.